src/harmony/MenuItem/MenuItem.tsx

```diff
@@ -22,18 +26,22 @@ export const MenuItem = ({
     children,
     size = 's',
     selectable,
+    multiple,
     ...props
 }: MenuItemProps) => {
+    const icon = nullable(
+        multiple,
+        () => <IconTickOutline size={size} />,
+        <Dot className={cn({ [s.MenuItemSelectedDot]: selected })} />,
+    );
+
     return (
         <div className={cn(s.MenuItem, { [s.MenuItem_hovered]: hovered }, className)} {...props}>
             {nullable(
                 selectable && !iconLeft,
-                () =>
-                    nullable(
-                        selected,
-                        () => <IconTickOutline size={size} />,
-                        <BaseIcon value={() => null} size={size} />,
-                    ),
+                () => (
+                    <BaseIcon value={() => nullable(selected, () => icon)} size={multiple ? size : dotSize} />
```

Comment:
Why u need BaseIcon here?

Reply:
To ensure that elements without icons are aligned relative to elements with icons

Reply:
BaseIcon is the only one way to align elements in MenuItem?

Reply:
each MenuItem element does not know which item is selected and with which icon (multiple or not)

baseIcon ensures the correct indentation